modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -1306,6 +1347,147 @@ public void testCorrectPartitionRecoveryOnSnapshot() throws InterruptedException
         }
     }
 
+    @Test
+    public void tableRecoveryOnMultipleRestartingNodesTest() throws InterruptedException {
```

Comment:
It's difficult to understand what do we check with given test. Please add corresponding javadoc.

Reply:
Yes, the one that existed before. Anyway, I've added a dedicated tests covering your scenarios.